### PR TITLE
MLPAB-1937 - Add access management instructions

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
# Purpose

Explain how to add users to a project

Fixes MLPAB-1937

### CODE REVIEW

File: docs/runbooks/setting_up_weblate.md [hunk 1]
```diff
@@ -36,3 +36,9 @@ Weblate will import the project, and show a Community Localisatoin Checklist. Yo
 
 1. Go to Settings in the repository in Github, and select Webhooks.
 2. Click "Add webhook" and set the Payload URL to `https://moj.weblate.cloud/hooks/github/`. Set the Content type to `application/x-www-form-urlencoded`. Don't set a secret, and select "Just Push Event" for the events to trigger the webhook. Click "Add webhook".
+
+### 4. Set project visibility in Weblate
```
Comment (andrewpearce-digital) [inline docs/runbooks/setting_up_weblate.md]:
```suggestion
### 4. Set up project access in Weblate
```

File: docs/runbooks/setting_up_weblate.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ When finished a project will be set up in the MoJ Weblate instance and the proje
 You will need to have the following:
 
 1. A Github account
-2. A Weblate account
+2. A Weblate account with Administrotor access
```
Comment (hawx) [inline docs/runbooks/setting_up_weblate.md]: Administrator

Comment (andrewpearce-digital) [inline docs/runbooks/setting_up_weblate.md]:
```suggestion
2. A Weblate account with Administrator access
```

Comment (andrewpearce-digital) [reply docs/runbooks/setting_up_weblate.md]: thanks, fixed

**Review by hawx — APPROVED** [review]

**Pull request merged**